src/test/java/stepdefinitions/MyStepdefs.java

```diff
@@ -93,6 +96,84 @@ public void theUserSeesOneProductInShoppingCart(final String EXP_Q) throws Inter
         org.junit.Assert.assertEquals(EXP_Q, amountOfProduct);
     }
 
+    @When("Get product list")
+    public void getProductList() {
+        homePage.getList();
+    }
+
+    @And("Check quantity equals expected")
+    public void checkQuantityEqualsExpected() {
+        actualSizeOfProductList = homePage.getSizeOfProductList();
+        Assert.assertEquals(Constants.EXP_PRODUCT_LIST_SIZE, actualSizeOfProductList);
+    }
+
+    @Then("Check that minimal quantity of product has expected price")
+    public void checkThatMinimalQuantityOfProductHasExpectedPrice() {
+        List<String> prices = homePage.getListOfPrice();
```

Comment:
Make this method reusable.
For example: Check that at least {int} of products have {string} price